.gitignore

```diff
@@ -30,7 +30,7 @@ lerna-debug.log
 
 # Environment variables files
 .env
-!samples/asgardeo-react-app/.env
+!samples/asgardeo-vite-app/.env
```

Comment:
Let's not change the app name. It's still a react app, we just use vite instead of webpack for bundling.

Reply:
then I delete the webpack project?

Reply:
Done